peregrine/stream_usrp.py

```diff
@@ -10,6 +10,7 @@
 from gnuradio import blocks
 from gnuradio import gr
 from gnuradio import uhd
+from gnuradio import gru
```

Comment:
How come there are edits to stream_usrp.py? Isn't this PR just for L2C code generation?